vllm/spec_decode/util.py

```diff
@@ -43,8 +43,8 @@ def get_sampled_token_logprobs(
                                        sampled_token_ids, ]
     expanded_selected_logprobs = selected_logprobs.unsqueeze(-1).expand(
         -1, -1, vocab_size)
-    sampled_token_ids_ranks = (logprob_tensor >=
-                               expanded_selected_logprobs).sum(-1)
+    sampled_token_ids_ranks = (logprob_tensor >
+                               expanded_selected_logprobs).sum(-1).add_(1)
```

Comment:
add a comment on why we add?

Reply:
In the regular decoding it was just like that as well. Since you already approved I did not change it so you don't  need to reapprove. 
But happy to add a comment of course if needed.